Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoftWindowsEventFirewall/README.md

```diff
@@ -0,0 +1,15 @@
+# Windows Events Firewall ASIM NetworkSessions Normalization Parser
+
+This template deploys the ASIM NetworkSessions schema parser for Windows Events Firewall. The parser is a part of the Advanced SIEM Information Model.
```

Comment:
"Windows Events Firewall" sounds bad. I think "Windows Firewall Events"?

Reply:
I concur. Again, changing labels is the easy part.